ee_tests/README.md

```diff
@@ -85,7 +84,20 @@ SELENIUM_BROWSER=firefox
 
 ### Typescript specs ###
 
-Run using the `ts-protractor.sh` helper script
+### setup ###
+
+```
+npm install -g yarn
+yarn
+
+```
+### run the tests ###
+
+Run using the `ts-protractor.sh [suite-name]` helper script
```

Comment:
why didn't you just add a `test` or `start` script to the package.json?

Reply:
I can/will when typescript based tests actually replace the old js tests. So far that isn't the case. 

I also think one less indirection may be better .. `npm test` -> `ts-protractor.sh` -> `protractor` but I agree following the convention is more important.

Reply:
one other thing is that .. if you want to pass additional arguments to the ts-protractor like the name of the test-suite, then the syntax isn't intuitive as the options need to be after a `--` e.g
`npm test -- --option-1 --option-2 test-suite`

Reply:
@joshuawilson  I made those proposed changes as well to start the test by running `npm start`